src/lib/dispatch/.may_include

```diff
@@ -0,0 +1,10 @@
+orconfig.h
+siphash.h
+ht.h
+
+lib/cc/*.h
+lib/container/*.h
+lib/dispatch/*.h
+lib/log/*.h
+lib/malloc/*.h
+lib/string/*.h
```

Comment:
nit: missing EOL

Reply:
Fixed in 8240d16c0375e836c42c98448fd6171156039fac